src/routes/visitsRouter.js

```diff
@@ -10,5 +10,6 @@ module.exports = function visitsRouter() {
     express.Router()
       .get('/', visitsController.get)
       .post('/', bodyVisitsValidator.addValidations, bodyVisitsValidator.validate, visitsController.add)
+      .post('/addExitTimestamp', bodyVisitsValidator.addExitTimestampValidations, bodyVisitsValidator.validate, visitsController.addExitTimestamp)
```

Comment:
Capaz algo mas REST seria un `patch('/', ...)`?

Reply:
Tuve la misma duda, pero como este endpoint puede o modificar una visita existente o crear una visita nueva si no existe estaba entre POST y PATCH. En [wikipedia](https://en.wikipedia.org/wiki/Patch_verb) dice `If the outcome of using the PATCH method is not known then the POST method is preferred.`, pero no me queda del todo claro